### ISSUE TITLE
Website isn't online

### ISSUE THREAD

**Issue by alisondoyle** [alisondoyle/css-validation-fixes#1]
My website is live but Markbot is telling me that it's not.

@thomasjbradley

thanks!

**Comment by thomasjbradley** [comment]
Have a look at the list of files on GitHub’s website; do they all follow our naming conventions?

**Comment by alisondoyle** [comment]
It doesn't on github, the index is capitalized but on atom it's not

Sent from my iPhone

On Nov 27, 2018, at 11:05 AM, Thomas J Bradley <notifications@github.com<mailto:notifications@github.com>> wrote:


Have a look at the list of files on GitHub’s website; do they all follow our naming conventions?

**Comment by thomasjbradley** [comment]
Yep. Because MacOS doesn’t distinguish between upper & lower case letters. Even though you changed it on your computer the change was never detected & therefore never committed. 

It’s easiest to make the change on GitHub’s website. Click the filename, then you’ll see a little “Edit” pencil to press. You can then rename the file.

**Comment by alisondoyle** [comment]
Awesome, thanks!

Sent from my iPhone

On Nov 27, 2018, at 11:21 AM, Thomas J Bradley <notifications@github.com<mailto:notifications@github.com>> wrote:


Yep. Because MacOS doesn’t distinguish between upper & lower case letters. Even though you changed it on your computer the change was never detected & therefore never committed.

It’s easiest to make the change on GitHub’s website. Click the filename, then you’ll see a little “Edit” pencil to press. You can then rename the file.
